src/core/control/Control.h

```diff
@@ -185,6 +186,11 @@ class Control:
     void startDragDropToolbar();
     bool isInDragAndDropToolbar();
 
+    /**
+     * Loads the view mode (hide/show menu-,tool-&sidebar)
+     */
+    bool loadViewMode(size_t mode);
```

Comment:
Why not use `enum ViewMode` as argument here? Comment above is not need - function name says it all.

Reply:
A future PR could add the possibility to be able to add custom modes. In that case the method should accept any mode-id (not only the default-modes defined in `enum ViewMode`).

But this gets refactored in my next commit anyway.